### ISSUE TITLE
Eclipse plugins ver. arc-2020.03-rc1 do not apply.

### ISSUE THREAD

**Issue by EvgeniiDidin** [foss-for-synopsys-dwc-arc-processors/toolchain#253]
Applying eclipse plugins in IDE fails with error:
```
requires 'osgi.bundle; org.eclipse.cdt.core 6.9.0' but it coul`d not be found
```
These plugins were built from "master" branch of https://github.com/foss-for-synopsys-dwc-arc-processors/arc_gnu_eclipse repository, which seems to be broken.

Applying plugins built from "arc-2019.09" branch finishes without any errors.

**Comment by anthony-kolesov** [comment]
Master branch has updated versions of plugins with newer requirements, please try using Eclipse IDE of version at least 2019-09 - they should install into it.

**Comment by abrodkin** [comment]
Fixed now in `arc-2020.03-rc1` re-spin with update of the Eclipse and JRE.
